Automattic/wp-calypso
# Social: Change multi-connection feature flag to a site feature

## Proposed Changes

This change is to work in conjunction with D110794-code. Rather than using the environment flags, it checks the active features for the site, allowing us to enable the feature on a per site basis.

## Testing Instructions

* Apply D110794-code and sandbox the API
* Check that a site with the blog sticker can still create multiple connections for Facebook and Tumblr on the /marketing/connections screen
* Check that a site without the blog sticker has the current behaviour of a second connection replacing the first

## Pre-merge Checklist
